apps/api-gateway/src/chat/chat.service.ts

```diff
@@ -4,4 +4,8 @@ import { ClientProxy } from '@nestjs/microservices';
 @Injectable()
 export class ChatService {
   public constructor(@Inject('CHAT_SERVICE') private readonly chatClient: ClientProxy) {}
+
+  public createRoom(roomName: string) {
+    return this.chatClient.send({ cmd: 'create_room' }, { roomName });
+  }
```

Comment:
```suggestion
  public createRoom(roomName: string) {
    return this.chatClient.send(MESSAGE.CHAT.CREATE_ROOM, { roomName });
  }
```
shared > constants 에 만들어놓은 메세지 패턴 정의 파일에
사용하시는 cmd 메세지들을 상수화 하여서 사용하면 좋겠습니다!

Reply:
[6da2345](https://github.com/GrebbMe/grabbme-be/pull/35/commits/6da2345f90842a259fe09d2d7eb912c9376e2979)

위 커밋에 반영하였습니다!